python/ray/experimental/client/server/server.py

```diff
@@ -61,6 +67,24 @@ def _return_debug_cluster_info(self, request, context=None) -> str:
             raise TypeError("Unsupported cluster info type")
         return json.dumps(data)
 
+    def release(self, client_id: str, id: bytes) -> bool:
+        released = False
+        if client_id in self.object_refs:
+            if id in self.object_refs[client_id]:
+                del self.object_refs[client_id][id]
+                released = True
+        return released
+
+    def release_all(self, client_id):
+        count = 0
+        if client_id in self.object_refs:
+            ids = set(self.object_refs[client_id].keys())
+            for id in ids:
+                count += 1
+                del self.object_refs[client_id][id]
+            del self.object_refs[client_id]
```

Comment:
Is the loop needed? We could also del the entire dict at once.

Reply:
Yeah, that'd probably work just as well. Call it being thorough. There's only so much one can do to direct Python to GC.